src/main/java/ca/corefacility/bioinformatics/irida/model/sequenceFile/Fast5Object.java

```diff
@@ -103,7 +105,10 @@ private Fast5Type setType(SequenceFile file) {
 		try {
 			String extension = FilenameUtils.getExtension(getFile().getFileName());
 
-			if (file.isGzipped()) {
+			// Checks if file is where it should be before it checks if it is gzipped
+			if (IridaFiles.fileExists(file.getFile()) && file.isGzipped()) {
+				type = Fast5Object.Fast5Type.ZIPPED;
+			} else if (extension.equals("gz")) {
 				type = Fast5Object.Fast5Type.ZIPPED;

```

Comment:
Got 2 parts of this if resulting in the same thing `ZIPPED`.  Should the extension checking be added as an `||` to the top condition?

Reply:
Yup I can't believe I did that. Updated in 117eedc7f06e586f08eed0fca9d520de62ca3c6f